n3fit/src/n3fit/tests/test_evolven3fit.py

```diff
@@ -23,11 +23,6 @@ def assert_sorted(arr, title):
         raise ValueError(f"The values of {title} are not sorted!")
 
 
-def check_consecutive_members(grid, value):
```

Comment:
Why did you remove this?

Reply:
Because now there are no consecutive members on the grids since they are separated by `nf`. So, the grid in Q itself contains only 4.92, but when a point belong to two different `nfs` the single point in Q will be added to each separated block.

Reply:
I see, fine then!